README.md

```diff
@@ -39,19 +59,22 @@ A location can be in the following forms:
 
 **2)** Clone and move inside the repo
 
-```
+```bash
+# Feel free to use https or ssh for cloning
 git clone https://github.com/oakestra/oakestra.git && cd oakestra
 ```
 
-**3)** Start the root and cluster orchestrator using the 1-DOC.yaml compose file.
+**3)** Start the root and cluster orchestrator using the `1-DOC.yaml` compose file.
 
-```
+```bash
 sudo -E docker-compose -f run-a-cluster/1-DOC.yaml up
+# If this should not work for you try "docker compose" instead
 ```
 
-1-DOC stands for 1 Device One Cluster, meaning that all the components are deployed locally. 
+**1-DOC** stands for 1 Device One Cluster, meaning that all the components are deployed locally. 
 
 ### Your first worker node 🍃
+<a name="your-first-worker-node 🍃"></a>
```

Comment:
I think there is a broken link here due to a missing `-`before 🍃

Reply:
Nicely spotted!
Funny enough it still worked for me without the "-" at the end. 
I have now added it.